client/src/main/java/io/pravega/client/stream/AckFuture.java

```diff
@@ -1,25 +0,0 @@
-/**
- * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- */
-package io.pravega.client.stream;
-
-import java.util.concurrent.Executor;
-import java.util.concurrent.Future;
-
-public interface AckFuture extends Future<Void> {
```

Comment:
Why is this change being done here? It is hard to track API-level changes like this if we piggyback them in some other change. This change is about a thread pool, right?